index.html

```diff
@@ -387,6 +388,71 @@ <h2>Citation</h2>
     </div>
 </div>
 
+<!-- Contact Us -->
+<div class="container" style="max-width: 768px">
+    <div class="row">
+        <div class="col-md-12">
+            <h2>Contact Us</h2>
+            <div class="row">
+                <div class="col-md-12">
+                    <form name="submit-to-google-sheet">
+                        <input type="text" name="Name" placeholder="Your Name" required>
+                        <input type="Email" name="Email" placeholder="Your Email" required>
+                        <textarea name="Message" rows="6" placeholder="Your Message"></textarea>
+                        <button type="submit" class="btn-secondary btn-secondary2">Submit</button>
```

Comment:
Once we click "submit" is it possible to show some animation to suggest it is being submitted. 
Also, when I tried to submit, I didn't get any confirmation that my submission was "successful".